config.json

```diff
@@ -358,6 +358,17 @@
         "control_flow_loops",
         "text_formatting"
       ]
+    },
+    {
+      "slug": "acronym",
+      "uuid": "54e60174-4833-418c-b9af-2fcb5722b1d6",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
Do you consider it the simplest `strings` or `filtering` exercise? If so, should we let a similar one be unlocked by this?

Reply:
Let's do this in another PR: We have a few more "floating" exercises that we should attach somewhere.